assets/migrations/mysql/002_add_authorization_model_version.sql

```diff
@@ -1,5 +1,5 @@
 -- +goose Up
-ALTER TABLE authorization_model ADD COLUMN schema_version INT NOT NULL DEFAULT 1;
+ALTER TABLE authorization_model ADD COLUMN schema_version VARCHAR(5) NOT NULL DEFAULT '1.0';
```

Comment:
VARCHAR(5) doesn't give us a lot of room for growth. Is it worth making it a little more generous?